src/components/MyCards/MyCardsActions.js

```diff
@@ -13,25 +18,33 @@ export const fetchCardsFailure = error => ({
     error
 });
 
-export const setPlayCard = playCard => ({
-    type: types.SET_PLAY_CARD,
-    playCard
-});
-
-export const selectCard = cardId => ({
-    type: types.SELECT_CARD,
+export const selectCardSuccess = cardId => ({
+    type: types.SELECT_CARD_SUCCESS,
     cardId
 });
 
-export const finishPlayCard = (cardId) => ({
-    type: types.FINISH_PLAY_CARD,
+export const removeCard = cardId => ({
+    type: types.REMOVE_CARD,
     cardId
 });
 
-export const resetFinishRound = finishedRound => ({
-    type: types.RESET_FINISH_ROUND,
-    finishedRound
-});
+export const selectCard = cardId => (dispatch, getState) => {
+    const status = getState().dashboardReducer.status;
+    if (status === "WAITING_FOR_CURRENT_PLAYER") {
+        dispatch(selectCardSuccess(cardId));
+    } else if (status === "WAITING_FOR_OTHER_PLAYERS") {
+        instance.post('/api/playCard', { cardId })
```

Comment:
You may, at some point, want to rename this variable as it's not very descriptive. Also, looks like it's a `let` that could be a `const`.

Reply:
Which variable, instance?

Reply:
Whoops, yes, I was referring to instance.

Reply:
Have changed it to a const, what would you rename it to? I think `instance` was just what they called it in whatever examples were found

Reply:
Even if you just called `axiosInstance` I think it would be more readable.